gp-includes/formats/format_properties.php

```diff
@@ -0,0 +1,500 @@
+<?php
+
+class GP_Format_Properties extends GP_Format {
+
+	public $name = 'Java Properties File (.properties)';
+	public $extension = 'properties';
+	public $filename_pattern = '%s_%s';
+
+	public $exported = '';
+
+	/**
+	 * Generates a string the contains the $entries to export in the Properties file format.
+	 *
+	 * @since 1.1.0
+	 *
+	 * @param GP_Project $project The project the strings are being exported for, not used
+	 *                            in this format but part of the scaffold of the parent object.
+	 * @param GP_Locale $locale The locale object the strings are being exported for. not used
+	 *                          in this format but part of the scaffold of the parent object.
+	 * @param GP_Translation_Set $translation_set The locale object the strings are being 
+	 *                                            exported for. not used in this format but part
+	 *                                            of the scaffold of the parent object.
+	 * @param GP_Translation $entries The entries to export.
+	 *
+	 * @return string
+	 */
+	public function print_exported_file( $project, $locale, $translation_set, $entries ) {
+		$result = '';
+
+		$sorted_entries = $entries;
+		usort( $sorted_entries, array( $this, 'sort_entries' ) );
+
+		foreach ( $sorted_entries as $entry ) {
+			$entry->context = $this->escape( $entry->context );
+			if ( empty( $entry->translations ) ) {
+				$translation = $entry->context;
+			} else {
+				$translation = $entry->translations[0];
+			}
+
+			$translation = str_replace( "\n", "\\n", $translation );
+			$translation = $this->utf8_uni_encode( $translation );
+
+			if ( empty( $entry->context ) ) {
+				$original = $entry->singular;
```

Comment:
Yes, it supports it.

It's not very pleasant but there's nothing wrong with the exported line according to the standard, it's just not very programmer friendly to type in.